sdk/tests/utilities/test_api_client_factory.py

```diff
@@ -44,6 +44,16 @@ def __getattribute__(self, name):
 
 
 class ApiFactory(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.os_environ_dict_str = 'os.environ'
+
+    def get_env_vars_wout_pat(self):
```

Comment:
wout? i presume you mean "without"?

Reply:
Updated.

Reply:
did you accidentally miss a commit? latest version still has `wout`